test-specifications/crypto-service/accountCreateTransaction.md

```diff
@@ -254,8 +254,9 @@ https://docs.hedera.com/hedera/sdks-and-apis/rest-api
 | Test no | Name                                                                               | Input                                          | Expected response                                                                                                                | Implemented (Y/N) |
 |---------|------------------------------------------------------------------------------------|------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------|
 | 1       | Creates an account with a max token association set to 0                           | key=<VALID_KEY>, maxAutoTokenAssociations=0    | The account creation succeeds and the account has 0 automatic token associations.                                                | N                 |
-| 2       | Creates an account with a max token association that is the maximum value          | key=<VALID_KEY>, maxAutoTokenAssociations=1000 | The account creation succeeds and the account has 1000 automatic token associations.                                             | N                 |
-| 3       | Creates an account with a max token association that is the maximum value plus one | key=<VALID_KEY>, maxAutoTokenAssociations=1001 | The account creation fails with a REQUESTED_NUM_AUTOMATIC_ASSOCIATIONS_EXCEEDS_ASSOCIATION_LIMIT response code from the network. | N                 |
+| 2       | Creates an account with a max token association set to 100                         | key=<VALID_KEY>, maxAutoTokenAssociations=100  | The account creation succeeds and the account has 0 automatic token associations.                                                | N                 |
```

Comment:
Should it have 100 automatic token associations (not 0)?
I would also propose same order as in memo (valid, min, max, invalid) 🙏🏻

Reply:
Agreed, updated the order and language a bit